### PR TITLE
SQLite plugin now checks if table exists before using the default one.

### PR DESCRIPTION
**Pull request by InFog**
It will check for tables named `rows_N`, no more only `rows`

### CODE REVIEW

**Comment by turicas** [comment]
@InFog, thanks for your PR! There are some things missing but I'll work on it:

- [ ] Create an automated test for this new feature
- [ ] Add yourself to `AUTHORS.md`
- [ ] Refactor the name creation function (could use `rows.plugins.utils._make_new_field_name`)

File: rows/plugins/sqlite.py [hunk 1]
```diff
@@ -64,6 +64,23 @@ def _get_connection(filename_or_connection):
...
+    connection = _get_connection(filename_or_connection)
+    select_tables = "SELECT name FROM sqlite_master WHERE type='table';"
+    current_tables = (x[0] for x in connection.execute(select_tables).fetchall())
+    table_name_raw = 'rows_{}'
+
+    i = 1
+
+    while True:
+        table_name = table_name_raw.format(i)
+        if not table_name in current_tables
```
Comment (turicas) [inline rows/plugins/sqlite.py]: `SyntaxError` here (missing comma).
You can check these errors by running the tests with:

    make test

**Comment by turicas** [comment]
@InFog, done!

Before merging into `develop` I added the following commits after yours:
- 444b95cb240276b002a6946f7a1fd39df911253c
- d476fb86222511cbd5d6900a17fc92a45bfd3eae

And also a merge with `develop` branch to solve the conflicts.

Thanks. :-)

**Comment by InFog** [comment]
Nice. Thanks for merging. I'll take more care for the next PRs ;)

**Pull request merged**